extensions/catalogs/accessing-private-registries.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="accessing-private-registries"]
+= Accessing private registries
+include::_attributes/common-attributes.adoc[]
+:context: accessing-private-registires
+
+toc::[]
+
+// Exclude from OSD/ROSA - dedicated-admins can't create the necessary secrets to do this procedure.
+
+If certain images relevant to Operators managed by Operator Lifecycle Manager (OLM) are hosted in an authenticated container image registry, also known as a private registry, OLM and OperatorHub are unable to pull the images by default. To enable access, you can create a pull secret that contains the authentication credentials for the registry. By referencing one or more pull secrets in a catalog source, OLM can handle placing the secrets in the Operator and catalog namespace to allow installation.
+
+Other images required by an Operator or its Operands might require access to private registries as well. OLM does not handle placing the secrets in target tenant namespaces for this scenario, but authentication credentials can be added to the global cluster pull secret or individual namespace service accounts to enable the required access.
+
+The following types of images should be considered when determining whether Operators managed by OLM have appropriate pull access:
```

Comment:
Didn't mean to actually include this (wasn't in the topic_map), because this feature isn't in v1 yet. Removing from PR, but will use these suggestions when the feature is added.